app/helpers/application_helper.rb

```diff
@@ -160,11 +160,20 @@ def translation(key, options = {}, html = true)
     translated_string2 = t(key, options)
 
     if html && current_user&.has_tag('translation-helper') && translated_string2.include?("translation missing") && !translated_string.include?("<")
-      raw(%(<span>#{translated_string} <a href="https://www.transifex.com/publiclab/publiclaborg/translate/#de/$?q=text%3A#{translated_string}">
+      raw(%(<span>#{translated_string} <a style='padding-left:3px;' href="https://www.transifex.com/publiclab/publiclaborg/translate/#de/$?q=text%3A#{translated_string}">
           <i data-toggle='tooltip' data-placement='top' title='Needs translation? Click to help translate this text.' style='position:relative; right:2px; color:#bbb; font-size: 15px;' class='fa fa-globe'></i></a>
        </span>))
     else
       raw(translated_string)
     end
   end
+
+  def create_dropdown(href, text)
```

Comment:
Ah, this is much cleaner. Could we rename this though so it's a little more specific? Let me think --- 
```suggestion
  def create_nav_dropdown_item(href, text)
```

Reply:
OK awesome. I think this is one of the last things before we merge, right? Thank you!!

Reply:
Yeah